### PR TITLE
Request Timeout Access Function

### PR DESCRIPTION
**Pull request by fpetrini15**
Server: https://github.com/triton-inference-server/server/pull/6483
Python Backend: https://github.com/triton-inference-server/python_backend/pull/315

### CODE REVIEW

File: include/triton/core/tritonbackend.h [hunk 1]
```diff
@@ -411,6 +411,16 @@ TRITONBACKEND_ResponseFactoryIsCancelled(
 TRITONBACKEND_DECLSPEC TRITONSERVER_Error* TRITONBACKEND_RequestCorrelationId(
     TRITONBACKEND_Request* request, uint64_t* id);
 
+/// Extract the specified timeout value from the request. If no timeout is
```
Comment (nnshah1) [inline include/triton/core/tritonbackend.h:414]:
```suggestion
/// Get the timeout of the request. A value of 0 indicates no timeout was specified for the request.
```

Comment (nnshah1) [reply include/triton/core/tritonbackend.h:414]: Trying to make a little more consistent with the description of other fields.
Can you also add a ticket to ensure the additional api is added to the bindings (Python / Java)

**Review by Tabrizian — COMMENTED** [review]
Other than @nnshah1's comment it looks good to me.

File: include/triton/core/tritonbackend.h [hunk 2]
```diff
@@ -411,6 +411,16 @@ TRITONBACKEND_ResponseFactoryIsCancelled(
 TRITONBACKEND_DECLSPEC TRITONSERVER_Error* TRITONBACKEND_RequestCorrelationId(
     TRITONBACKEND_Request* request, uint64_t* id);
 
+/// Get the timeout of the request. A value of 0 indicates no timeout was specified for the request.
+/// provided, the timeout will default to zero.
```
Comment (Tabrizian) [inline include/triton/core/tritonbackend.h:415]: Please keep the number of characters in each line below 80.

File: include/triton/core/tritonbackend.h (hunk 1 above)
Comment (fpetrini15) [reply include/triton/core/tritonbackend.h]: Added DLIS-5869

**Review by Tabrizian — APPROVED** [review]

**Pull request merged**